app/views/users/show.html.erb

```diff
@@ -1,3 +1,40 @@
-<h1>
-  this is the detail page for the user with id
-</h1>
\ No newline at end of file
+<header>
+  <%= stylesheet_link_tag 'user', media: 'all', 'data-turbolinks-track': 'reload' %>
+</header>
+
+<body>
+  <%= render partial: "partials/users", locals: { user: @user } %>
+
+  <div id="bio">
+    Bio: <%= @user.bio %>
+  </div>
+
+  <div id="singlePost">
+    <h2>Posts</h2>
+    <ul>
+      <% @recent_posts.each do |post| %>
+        <li>
+          <%= link_to post.title, user_post_path(@user, post) %>
+          <span class="post-details">
+            Likes: <%= post.likes_counter %>
+            Comments: <%= post.comments_counter %>
+          </span>
+
+          <div class="comments-container" id="comments_for_post_<%= post.id %>">
+            <h4>Comments:</h4>
+            <ul>
+              <% post.comments.each do |comment| %>
+                <li><%= comment.text %></li>
+              <% end %>
+            </ul>
+          </div>
+        </li>
```

Comment:
- [ ] An error occurs when I try to click a post. The correct behaviour is for the post show page to be loaded. Please see the error below and try to fix it
![image](https://github.com/AleWaweru/blog-App/assets/46839548/b6bf041b-6489-4271-8d11-a34ae076fc09)

Reply:
Resolved